### DISCUSSION TITLE
Classic thumbnails using Programmatic code

### DISCUSSION THREAD

**Discussion by mattie2dope** [fancyapps/ui#577] [Q&A] [answered]
Hi,
I’m sure this is something super simple but for the life of me I can’t figure it out.
Using the HTML, Programmatic, [Video gallery with thumbnails](https://jsfiddle.net/yrp254k9/) example how do you get Classic thumbnails to work?
I have tried pasting 

const options = {
  Thumbs: {
    type: "classic",
  },
};

in pretty much every possible location but it either breaks it or doesn't work.
Thanks for your time!

**Comment by fancyapps** [comment, marked as answer]
Hi,

First, include corresponding JS and CSS files, then apply gallery options, like this:

```js
Fancybox.show([
  {
    src: "https://www.youtube.com/watch?v=z2X2HaTvkl8",
    thumb: "http://i3.ytimg.com/vi/z2X2HaTvkl8/hqdefault.jpg",
  },
  {
    src: "https://www.youtube.com/watch?v=dZRqB0JLizw",
    thumb: "http://i3.ytimg.com/vi/dZRqB0JLizw/hqdefault.jpg",
  },
  {
    src: "https://vimeo.com/259411563",
    thumb: "https://f.vimeocdn.com/images_v6/lohp/video1_thumbnail.png",
  },
],{
  Thumbs: {
    type: "classic",
  }
}); 
```

https://jsfiddle.net/wsvz57pr/

**Reply by mattie2dope** [reply]
Brilliant,
Works perfectly!
Thank you so much for your help on this, it is greatly appreciated!
